lib/tessel_2/driver.js

```diff
@@ -151,12 +155,34 @@ GPS.prototype._emitCoordinates = function (parsed) {
 
       latitude = [latDeg, latMin, latSec, latPole];
       longitude = [lonDeg, lonMin, lonSec, lonPole];
-    } else if (self.format === 'deg-dec') {
+    } else if (self.options.format === 'deg-dec') {
       lat = latDeg + (latMin / 60);
       lon = lonDeg + (lonMin / 60);
 
       latitude = [lat, latPole];
       longitude = [lon, lonPole];
+    } else if (self.options.format === 'utm') {
+      // Convert to degree notation
```

Comment:
The only code it could reuse is the two lines just below (I started implementing it that way) but it makes the code less readable that way.